core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -65,6 +92,7 @@ java.util.LinkedHashMap
 java.util.LinkedHashSet
 java.util.LinkedList
 java.util.Locale
+java.util.Optional
```

Comment:
Cf. https://github.com/jenkinsci/octoperf-plugin/commit/12ffdc2ddc9dfbb44f784aae4a9bea5eac1e8329; probably ought to have been a simple nullable field.

Reply:
Seems to be a harmless fix in general.
It may be also reasonable for other JVM-based languages which are null-safe, just in case somebody writes plugins in them